src/super_gradients/recipes/dataset_params/cifar100_dataset_params.yaml

```diff
@@ -0,0 +1,54 @@
+-batch_size: 256 # batch size for trainset
+-val_batch_size: 512 # batch size for valset in DatasetInterface
+
+# TODO: REMOVE ABOVE, HERE FOR COMPATIBILITY UNTIL WE REMOVE DATASET_INTERFACE
+
+train_dataset_params:
+  root: /data/cifar100
+  train: True
+  transform:
```

Comment:
It would look nicer if we align the "transform" arg as done in the other yaml files (i.e transforms).
This can be done if we reimplement the cifar10/100 datasets (that would inherit from the original ones of course) and resolve transforms with ListFactory(TransformsFactory()).

And while also speaking of resolving the transforms- lets take out the _target_ and use the factories for that.